### DISCUSSION TITLE
Pins Don't Appear In Start

### DISCUSSION THREAD

**Discussion by Jetzarunie** [valinet/ExplorerPatcher#4037] [General]
For me when I try to pin anything to start on the windows 10 start menu on any of the three options none of them show up.

https://github.com/user-attachments/assets/2da5ce8c-7900-4620-8bc8-e2cdb0cf651c

![image](https://github.com/user-attachments/assets/0e9d98d8-7d60-4e8a-90a2-8570fc755287)

![image](https://github.com/user-attachments/assets/bdfc49af-e584-48c8-b37b-8461604764f9)

**Comment by pyrates999** [comment]
For windows 10 start menu related issues, you can discuss it here: https://github.com/valinet/ExplorerPatcher/discussions/1679
